python/modules/Stop0lObjectsProducer.py

```diff
@@ -133,21 +133,23 @@ def SelSoftb(self, isv):
 
 
     def SelJets(self, jet):
-        if jet.pt < 20 or math.fabs(jet.eta) > 2.4 :
+        if jet.pt < 20 or math.fabs(jet.eta) > 2.4:
             return False
         return True
 
     def SelPhotons(self, photon):
-        if photon.pt < 200:
+        if photon.pt < 220:
             return False
         abeta = math.fabs(photon.eta)
-        if (abeta > 1.442 and abeta < 1.566) or (abeta > 2.5):
+        if (abeta > 1.4442 and abeta < 1.566) or (abeta > 2.5):
             return False
-        ## cut-base ID, 2^0 loose ID
-        cutbase =  photon.cutBasedBitmap  if self.era != "2016" else photon.cutBased
-        if not cutbase & 0b1:
-            return False
-        return True
+        # --- Photon ID: cut-based medium ID --- #
```

Comment:
So medium ID is OK for Zinv? Enough statistics?

Reply:
In my initial testing of the medium ID, there are enough statistics.

Reply:
But your implementation here is meant for V3. Can you confirm this is the ID you will use eventually?

Reply:
I am currently running jobs with jet pt > 20 GeV and medium photon ID. Maybe next week we can see my results and decide on the ID.

Reply:
OK